### PR TITLE
Add Score Progress bar to Roses 

### PR DESCRIPTION
**Pull request by danleedham**
I think I've finally built something that's finished enough to eb added. 

User Toggles on Winning Progress which presents a bar on the left hand side . 

Displays nothing when at zero, but as we'll be streaming after some points have already scored that's academic. 

Works will josn and manual scores. 

screen shot 2018-04-10 at 20 10 40

Tested on Caspar Server Beta 2.1.0. Not tested with specific video underneath so might want to change the color of the the 177.5, or pop it in a box etc.

### CODE REVIEW

File: public/js/cg.js [hunk 1]
```diff
@@ -140,14 +140,22 @@ app.controller('scoringCtrl', ['$scope', '$interval', '$http', 'socket',
...
             $scope.showScore = state.showScore;
             $scope.manualScore = state.manualScore;
+            $scope.showProgress = state.showProgress;
             if(state.manualScore){
               $scope.yorkScore = state.yorkScore;
               $scope.lancScore = state.lancScore;
             };
+            
+            var available = $scope.yorkScore + $scope.lancScore;
+			var toWin = 177.5;
```
Comment (stephenrob) [inline public/js/cg.js]: Could we extract this out to a constant at the top making this easier to change? Might also be worth making note of the total points available as well.

**Review by stephenrob — CHANGES_REQUESTED** [review]
Location of 'points to win' definition to be clearer and easy to edit.

**Comment by danleedham** [comment]
Is the the points to win likely to change across the weekend? 

I had thought of using the fixtures data to count all the points available, then do the math but was worried that could lead to errors if the data wasn't perfect. 

The available variable doesn't actually do anything, it was just if I wanted to use the current amount of points currently scored. I'll remove it for the absence of doubt. 

Is the total points available not always `(toWin x2 ) - 1` ?

**Review by stephenrob — APPROVED** [review]

**Comment by stephenrob** [comment]
We can look at possibly pulling the total points available from roseslive once i've evaluated the code but for now this can be merged in.

**Pull request merged**